compiler/testData/codegen/bytecodeText/inlineClasses/toStringIsCalledByInlineClass.kt

```diff
@@ -1,13 +1,25 @@
 // !LANGUAGE: +InlineClasses
+// Completely incorrect bytecode in some of these - see `box/inlineClasses/toStringOfUnboxedNullable.kt`
+// IGNORE_BACKEND: JVM
```

Comment:
Please extract the parts that do work in JVM, into a separate non-muted test

Reply:
Moved the class that wraps `Any` to a separate test